resources/views/account/partials/member-admin-action-bar.blade.php

```diff
@@ -10,307 +10,284 @@
                     </a>
                 </h3>
             </div>
-            <div id="collapseMemberAdminActions" class="panel-collapse collapse in" role="tabpanel">
-                <div class="panel-body" style="background: repeating-linear-gradient( 45deg, #fafafa, #fafafa 40px, #fff 40px, #fff 80px )">
-                    <div class="row"> <!-- Start Row -->
-                        <div class="col-md-12">
-                            <li>
-                                <b>Name:</b> {{ $user->given_name }} {{ $user->family_name }}
-                            </li>
-                            <li>
-                                <b>Email:</b> {{ $user->email }} 
-                                <small>(
-                                    @if($user->email_verified)
-                                        Email verified
-                                    @else
-                                        Email NOT verified
-                                    @endif
-                                )
-                                </small>
-                            </li>
-                            <li>
-                                <b>Last Seen:</b> {{ $user->seen_at }}
-                            </li>
-                        </div>
-                        <!-- Panel -->
-                        <div class="col-md-6" style="border-left: 3px solid green;">
-                            <h4>Trusted Member</h4>
-                            <p>The member will be automatically emailed about being made trusted but not if they are loosing trusted status.</p>
-                            {!! Form::open(array('method'=>'PUT', 'route' => ['account.admin-update', $user->id], 'class'=>'form-horizontal')) !!}
-                            <div class="form-group">
-                                <div class="col-sm-5">
-                                @if ($user->trusted)
-                                    {!! Form::hidden('trusted', 0) !!}
-                                    {!! Form::submit('Remove Trusted Status', array('class'=>'btn btn-default')) !!}
-                                @else
-                                    {!! Form::hidden('trusted', 1) !!}
-                                    {!! Form::submit('Make a trusted member', array('class'=>'btn btn-default')) !!}
-                                @endif
-                                </div>
-                            </div>
-                            {!! Form::close() !!}
+            <div class="infobox infobox__grid" role="tabpanel">
```

Comment:
Could we bring back `id="collapseMemberAdminActions"`; it lets us collapse the entire "Admin actions" block if we don't want to see them (can always expand again)

```suggestion
            <div id="collapseMemberAdminActions" class="infobox infobox__grid" role="tabpanel">
```